classes/QueryFormatter.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Queries report class
+ *
+ * @since 3.0.0
+ * @package DebugBarElasticPress
+ */
+
+namespace DebugBarElasticPress;
+
+use \ElasticPress\QueryLogger;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Queries report class
+ *
+ * @package DebugBarElasticPress
+ */
+class QueryFormatter extends QueryLogger {
+	/**
+	 * Format queries to display
+	 *
+	 * @param array $queries Queries to be displayed
+	 * @return array
+	 */
+	public function format_queries_for_display( $queries ) : array {
+		$formatted_queries = [];
+
+		$labels = [
+			'wp_url'      => esc_html__( 'Page URL', 'elasticpress' ),
```

Comment:
Invalid text-domain. It should be `debug-bar-elasticpress`

Reply:
Fixed in https://github.com/10up/debug-bar-elasticpress/commit/977885082578323404c91e430ab09a6f7d16a17a. Thanks @burhandodhy!